_targets.R

```diff
@@ -211,25 +229,37 @@ list(
       # first pass of flags
       all_data_flagged <- map(all_data_summary_stats_list, function(data) {
         data %>%
+          # sensor was handled by technician
           add_field_flag() %>%
+          # observation is outside sensor spec range
           add_spec_flag() %>%
+          # observation is outside seasonal 1-99 percentile range
           add_seasonal_flag() %>%
+          # no data
           add_na_flag() %>%
+          # repeating vale
           add_repeat_flag() %>%
+          # if >50% of data is flagged for any reason above in a rolling
+          # 3-hour window, flag all the data:
           add_suspect_flag() %>%
-          # we should also be incorporating the add_malfuntion_flag() here no?
-          mutate(mean_public = ifelse(is.na(flag), mean, NA)) #%>%
-        #mutate(historical_flagged_data_1 = TRUE)
+          # flag for known instances of sensor malfunction:
+          add_malfunction_flag() %>%
+          # store censored, clean data in new column
+          mutate(mean_public = ifelse(is.na(flag), mean, NA))
```

Comment:
to do: we're going to want to make sure that we are keeping track of how often data is being flagged. I think that's how I determined how to join the data back together after we reuse flagged data to generate the summary stats and reflag the data. If this does not make sense we should talk about it because I have not worked on that in a while and I don't remember if it's actually in place or not.

Reply:
this is essentially making sure that issue #73 is actually completed and working properly.